### DISCUSSION TITLE
Access API to render openPose images from folder

### DISCUSSION THREAD

**Discussion by rbaechtold** [vladmandic/sdnext#555] [Q&A]
Hi all
I'm trying to find some information to start the API with this fork. There is an option in automatic1111 to do this. My plan is to write a script (well, since I'm no developer chatGPT might do it) to batch process an openPose image folder so I can generate the previews for those. In order for the script to work, I would need to access the API with the URL given by automatic. Do you think this is possible? Sorry, I'm no developer ^^ I'd like to try to get this running nonetheless because as it seems, nobody has created this.
cheers

**Comment by vladmandic** [comment]
the same api docs as a1111 apply. eventually, i'll write docs here as well, just no time yet.
it should be possible, but its not trivial as openpose is a an extension, so it doesn't have a direct api endpoint, it should be part of "running scripts" inside api request.
